app/views/welcome/_next-event.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class='main'>
+  <h1> Next Event </h1>
+  <% if !next_event %>
+    <p>There are no upcoming events. <%= link_to 'Create one', new_event_path %>  to get the party started.<p>
+  <% else %>
+    <h2><%= next_event.name %></h2>
+    <h3><%= next_event.date.strftime("%B %d, %Y")%></h3>
+    <% if next_event.presentations.empty? %>
+      <p>There are currently no presentations for this event. <%= link_to 'Sign up', new_presentation_path %> to teach some people something!</p>
+    <% else %>
+      <p> Presentations from: </p>
+      <ul>
+        <% next_event.presentations.each do |presentation| %>
+          <li><span><%= presentation.presenter %> &ndash; <%= presentation.topic %> (<%=presentation.duration %> min)</span></li>
```

Comment:
Not sure how rubocop isn't barfing at this but you're over 80 char. I would try splitting this line up into a couple lines.